sunpy-dev-env.yml

```diff
@@ -53,6 +53,7 @@ dependencies:
   - tox-conda
 
   # Documentation
+  - astroquery
```

Comment:
What example still uses it?

Reply:
[Identifying stars in a STEREO/SECCHI COR2 coronagraph image](https://docs.sunpy.org/en/stable/generated/gallery/units_and_coordinates/STEREO_SECCHI_starfield.html)